prairielib/lib/sql-db.js

```diff
@@ -916,5 +916,6 @@ module.exports = {
   setSearchSchema: defaultPool.setSearchSchema,
   getSearchSchema: defaultPool.getSearchSchema,
   setRandomSearchSchemaAsync: defaultPool.setRandomSearchSchemaAsync,
-  PostgresPool,
 };
+
+module.exports.PostgresPool = PostgresPool;
```

Comment:
Just out of curiosity, was this change necessary?

Reply:
I'm not 100% sure *why*, but TypeScript didn't like it when I tried to do something like `/** @type import('./sql-db').PostgresPool */` with the old way 🤷